doc/browsing_cubes/intro.rst

```diff
@@ -1,9 +1,54 @@
 Browsing Cubes
 ==============
 
-This is a description of how you can use the cube_browser package to look at cubes and make plots and sliders without having to make any widgets.
+The Cube Browser package offers an API which you can use to visualize your cubes in a Jupyter notebook.
```

Comment:
In fact, we could even add a link to the Iris User Guide on [what Iris cubes are](http://scitools.org.uk/iris/docs/latest/userguide/iris_cubes.html) here as well.